doc/contributing.md

```diff
@@ -16,37 +16,217 @@ and we encourage all to read it carefully.
   indicate that they are beginner friendly, meaning that they don't require extensive
   knowledge of the project.
 * Make a tutorial or gallery example of how to do something.
+* Improve the API documentation.
 * Provide feedback about how we can improve the project or about your particular use
-  case.
-* Contribute code you already have. It doesn't need to be perfect! We will help you
-  clean things up, test it, etc.
+  case. Open an [issue](https://github.com/GenericMappingTools/pygmt/issues) with
+  feature requests or bug fixes, or post general comments/questions on the
+  [forum](https://forum.generic-mapping-tools.org).
+* Contribute code! This can be code that you already have and it doesn't need to be
+  perfect! We will help you clean things up, test it, etc.
+* Help triage issues, or give a "thumbs up" on issues that others reported which are
+  relevant to you.
+* Participate and answer questions on the [PyGMT forum Q&A](https://forum.generic-mapping-tools.org/c/questions/pygmt-q-a/11).
+* Participate in discussions at the quarterly PyGMT Community Meetings, which are
+  announced on the [forum governance page](https://forum.generic-mapping-tools.org/c/governance/9).
+* Spread the word about PyGMT or star the project!
```

Comment:
Great to have a more general summary of ways to contribute to PyGMT. Shall we use nested lists (see **Providing Feedback** & **General Guidelines for Making a Pull Request (PR)** for examples)? It seems that a list of about 3-5 terms is friendly. The first five terms seem to be related to code/documentation contribution and the last three about question & discussion.